aws/policy/compute.yaml

```diff
@@ -92,7 +92,9 @@ Statement:
   - Sid: AllowGlobalResourceRestrictedActionsWhichIncurNoFees
     Effect: Allow
     Action:
+      - ec2:AssignPrivateIpAddresses
```

Comment:
It's important to test these things before merging because neither of these supports resource restriction. https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazonec2.html
A shortcut to reading the documentation (that *usually* works but not always) is https://github.com/ansible/ansible/blob/devel/hacking/aws_config/build_iam_policy_framework.py